test/user.spec.js

```diff
@@ -0,0 +1,114 @@
+// import chai from 'chai';
```

Comment:
Why is this commented out?

Reply:
There was no need for it anymore since I could use just `request`.